azure-pipelines.yml

```diff
@@ -21,9 +20,10 @@ jobs:
     - bash: |
         source activate gudhi_build_env
         sudo conda install --yes --quiet --name gudhi_build_env python=$(pythonVersion)
+        sudo conda update --yes --quiet -n base -c defaults conda
+        sudo conda install --yes -c conda-forge doxygen cgal-cpp=5.0.1 eigen
```

Comment:
Would replacing `=` with `>=` work? It seems less likely to force us to update it soon.

Reply:
Yes you are right it will be less restrictive